api/candlepin-api-spec.yaml

```diff
@@ -1265,6 +1265,35 @@ components:
           type: string
           minLength: 1
 
+    EnvironmentDTO:
+      description: DTO representing an environment
```

Comment:
Since we're trying to have this spec also be a good source of documentation, we should provide better descriptions.

A good guideline for all descriptions is: Try to use information from the model entity's class javadoc comment (in this case Environment.java), instead of from the DTO, because our DTOs don't have useful documentation (it's always something like "A DTO representation of X entity"). Then, take that and try to distill it to not have any implementation details, and be concise. In this case it could be something like: "Represents an environment within an org used to enable/disable/promote content in specific places"

Reply:
Agreed. Added description for `EnvironmentDTO` & `EnvironmentContentDTO`